.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "files.associations": {
+    "*.mdx": "javascriptreact"
```

Comment:
It was frustrating loading up a `.mdx` file in here and have VSCode treat it as a plain Markdown file.